addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -291,23 +291,23 @@ var SlideUploadDialog = Dialog.extend({
                 }
             },
             fill_data: function (query, data) {
-                var that = this,
+                var self = this,
                     tags = {results: []};
                 _.each(data, function (obj) {
-                    if (that.matcher(query.term, obj[nameKey])) {
+                    if (self.matcher(query.term, obj[nameKey])) {
                         tags.results.push({id: obj.id, text: obj[nameKey]});
                     }
                 });
                 query.callback(tags);
             },
             query: function (query) {
-                var that = this;
+                var self = this;
                 // fetch data only once and store it
                 if (!this.selection_data) {
                     this.fetch_rpc_fnc().then(function (data) {
-                        that.can_create = data.can_create;
-                        that.fill_data(query, data.read_results);
-                        that.selection_data = data.read_results;
+                        self.can_create = data.can_create;
+                        self.fill_data(query, data.read_results);
+                        self.selection_data = data.read_results;
```

Comment:
Variables defined on self could be camelCased.

Reply:
I would keep it like this to avoid regressions ... until we decide to refactor slides JS.